src/routes/safe/components/Settings/Advanced/RemoveGuardModal.tsx

```diff
@@ -121,71 +60,28 @@ export const RemoveGuardModal = ({ onClose, guardAddress }: RemoveGuardModalProp
       title="Remove Transaction Guard"
       open
     >
-      <EditableTxParameters
-        isOffChainSignature={isOffChainSignature}
-        isExecution={canTxExecute}
-        ethGasLimit={gasLimit}
-        ethGasPrice={gasPriceFormatted}
-        safeTxGas={gasEstimation}
-        closeEditModalCallback={closeEditModalCallback}
-      >
-        {(txParameters, toggleEditMode) => {
-          return (
-            <>
-              <ModalHeader onClose={onClose} title="Remove Guard" />
-              <Hairline />
-              <Block>
-                <Row className={classes.modalOwner}>
-                  <Col align="center" xs={1}>
-                    <PrefixedEthHashInfo
-                      hash={guardAddress}
-                      showCopyBtn
-                      showAvatar
-                      explorerUrl={getExplorerInfo(guardAddress)}
-                    />
-                  </Col>
-                </Row>
-                <Row className={classes.modalDescription}>
-                  <Paragraph noMargin size="lg">
-                    Once the transaction guard has been removed, checks by the transaction guard will not be conducted
-                    before or after any subsequent transactions.
-                  </Paragraph>
-                </Row>
-              </Block>
-              <Block className={classes.accordionContainer}>
-                {/* Tx Parameters */}
-                <TxParametersDetail
-                  txParameters={txParameters}
-                  onEdit={toggleEditMode}
-                  isTransactionCreation={isCreation}
-                  isTransactionExecution={canTxExecute}
-                  isOffChainSignature={isOffChainSignature}
-                />
-              </Block>
-              <Row className={classes.modalDescription}>
-                <ReviewInfoText
-                  gasCostFormatted={gasCostFormatted}
-                  isCreation={isCreation}
-                  isExecution={canTxExecute}
-                  safeNonce={txParameters.safeNonce}
-                  txEstimationExecutionStatus={txEstimationExecutionStatus}
-                />
-              </Row>
-              <GenericModal.Footer withoutBorder={buttonStatus !== ButtonStatus.LOADING}>
-                <GenericModal.Footer.Buttons
-                  cancelButtonProps={{ onClick: onClose }}
-                  confirmButtonProps={{
-                    color: 'error',
-                    onClick: () => removeTransactionGuard(txParameters),
-                    status: buttonStatus,
-                    text: confirmButtonText,
-                  }}
-                />
-              </GenericModal.Footer>
-            </>
-          )
-        }}
-      </EditableTxParameters>
+      <TxParamsState txData={txData} txValue="0" onSubmit={removeTransactionGuard} submitText="Remove">
```

Comment:
Why is `txValue` set to 0 here?

Reply:
This one is outdated. I already removed it